### DISCUSSION TITLE
Where can I buy custom keycaps for the QWERTY-fr layout?

### DISCUSSION THREAD

**Discussion by yyewolf** [qwerty-fr/qwerty-fr#80] [Ideas]
Hi,

First of all, the layout seems amazing and I haven't had the occasion to try it because I'd prefer to try it with a keyboard that matches the layout.

Is there any plans to share a few links to some retailers that might sell qwerty-fr keycaps / keyboards ?

**Comment by devnoname120** [comment]
There have been some discussions on Telegram and one of our community members ordered custom keycaps exactly for that purpose :) I'd recommend you to join and ask there.
